distributed/comm/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import print_function, division, absolute_import
+
+from .core import connect, listen, CommClosedError
+# Register transports
+from . import tcp
+try:
+    import zmq as _zmq
+except ImportError:
+    pass
+else:
+    # XXX this registers the ZMQ event loop, event if ZMQ is unused...
+    from . import zmq
```

Comment:
I suggest that we extend `distributed.config` to also collect all environment variables named `DASK_FOO` and put `foo` into the global config dict.  We could then test ZeroMQ by changing environment variables (which is easy on travis-ci).

I guess we would need to special case the import, `if default_transport == 'zmq': import ...`.

Thoughts?